_posts/2022-6-16-how-disney-improved-activity-recognition-with-multimodal-approaches-with-pytorch.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog_detail
+title: "How Disney Improved Activity Recognition Through Multimodal Approaches with PyTorch"
+author: Monica Alfaro, Albert Aparicio, Francesc Guitart, Marc Junyent, Pablo Pernias, Marcel Porta, and Miquel Àngel Farré (former Senior Technology Manager)
+featured-img: '/assets/images/how-disney-improved-social.jpg'
+---
+
+# Introduction
```

Comment:
I think all headings need to be level 2 (##) like in other posts.

Reply:
Request was to set the headings as indicated inside the post. Headings do not need to be adjusted.